typings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "globalDependencies": {
+    "mocha": "registry:dt/mocha#2.2.5+20161028141524",
```

Comment:
The version numbers here don't match those in the file above (e.g. mocha 3.1.2 vs. 2.2.5 and sinon 1.17.6 vs. 1.16.0...)